### PR TITLE
Loadout cleanup and improvement

### PR DESCRIPTION
**Pull request by tf-4**
# This PR does not remove anything from the loadout. If you're suddenly not spawning with your usual loadout item(s), check the item isn't elsewhere in the list and that yours was just the duplicate. If it's fully removed, let me know as that is not intentional.

## About The Pull Request

Cleans up and organises the loadout datums.
Much of the great big fuck-off walls of datums are now split up with comments denoting section.
Capitalises each word in the datum, unless a word like "of" or whatever. Please follow this in the future.

Removes a few duplicate items. **Some of these were meant to be job restricted and had an unrestricted duplicate. The unrestricted variants have been removed in favour of the job-restricted ones.** This includes things like the sterile mask.

Adds `required_season`, a variable that hides the datum if it's not the appropriate season (currently only used for Christmas).

Adds **armourless** versions of the sinew skirt, bone talisman, and skull codpiece.
Adds **armourless** versions of the witch and wizard robes and hats.

## How This Contributes To The Skyrat Roleplay Experience

Better code good. Worse code bad. Having to _uncomment loadout items every Christmas_ bad.

There is more I want to come back to regarding the loadout and its organisation, but my eyes started to cross by the end of doing this and I also don't massively want to do it all in one PR anyway.

## Changelog
:cl:
add: Armourless versions of the sinew skirt, bone talisman, and skull codpiece are in the loadout.
add: Armourless versions of the witch and wizard robes and hats are in the loadout.
spellcheck: (Hopefully all) the loadout options are now properly capitalised.
code: You can now restrict loadout items to specific holidays (such as Christmas).
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 1]
```diff
@@ -18,20 +20,9 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	else
 		outfit.head = item_path
 
-
-//readd these on christmas or suffer my wrath
```
Comment (theselfish) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm:22]: Any reason these are being removed?

It'd be more annoying for someone to fully readd these around Christmas.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 2]
```diff
@@ -213,7 +224,28 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	name = "Bowler Hat"
 	item_path = /obj/item/clothing/head/bowler
 
-//HALLOWEEN
+/*
+*	CHRISTMAS
+*/
+
+/datum/loadout_item/head/santa
```
Comment (theselfish) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm:231]: I'm blind, ignore my last comment.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm [hunk 3]
```diff
@@ -46,10 +37,18 @@ GLOBAL_LIST_INIT(loadout_shoes, generate_loadout_items(/datum/loadout_item/shoes
...
+/*
+*	MISC BOOTS
+*/
+
 /datum/loadout_item/shoes/timbs
 	name = "Fashionable Boots"
 	item_path = /obj/item/clothing/shoes/jackboots/timbs
 
+/datum/loadout_item/shoes/jungle
+	name = "Brown Jackboots"
```
Comment (theselfish) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm]: Shouldn't these be under jackboots?

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm (hunk 1 above)
Comment (tf-4) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm:22]: They aren't.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm (hunk 3 above)
Comment (tf-4) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_shoes.dm]: I was ehhh on that as they're not actually jackboots. Hm. I should rename them to "Jungle Boots" though if I do that, yes.

**Pull request merged**